src/main/java/org/broadinstitute/hellbender/utils/runtime/CapturedStreamOutputSnapshot.java

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.hellbender.utils.runtime;
+
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.*;
+
+/**
+ * Stream output captured from a streaming stream.
+ */
+public final class CapturedStreamOutputSnapshot extends CapturedStreamOutput {
+
+    /**
+     * @param settings       Settings that define what to capture.
+     * @param processStream  Stream to capture output.
+     * @param standardStream Stream to write debug output.
+     */
+    public CapturedStreamOutputSnapshot(final OutputStreamSettings settings, final InputStream processStream, final PrintStream standardStream) {
+        super(settings, processStream, standardStream);
+
+        // strictly speaking, we could support these, but their use in streaming applications seems questionable
+        Utils.validate(!settings.getStreamLocations().contains(StreamLocation.Standard), "Stream snapshots can't go to standard out");
+        Utils.validate(!settings.getStreamLocations().contains(StreamLocation.File), "Stream snapshots can't go to a file");
+    }
+
+    /**
+     * Drain the input stream to keep the process from backing up until it's empty.
+     * File streams will be closed automatically when this method returns.
```

Comment:
is this true?  I don't see the close here anymore.

Reply:
this should describe what the return value means

Reply:
Yes this should be removed as its not true of this implementation.